homeassistant/components/rflink/sensor.py

```diff
@@ -116,7 +183,9 @@ def __init__(
     ):
         """Handle sensor specific args and super init."""
         self._sensor_type = sensor_type
-        self._unit_of_measurement = unit_of_measurement
+        self._attr_native_unit_of_measurement = unit_of_measurement
```

Comment:
But that in the case of not having an entity description

Reply:
The intention is that user can overwrite the sensor UOM, if not the EntityDescriptor will be taken, if no ED then no UOM.
```python
    @property
    def native_unit_of_measurement(self) -> str | None:
        """Return the unit of measurement of the sensor, if any."""
        if hasattr(self, "_attr_native_unit_of_measurement"):
            return self._attr_native_unit_of_measurement
        if hasattr(self, "entity_description"):
            return self.entity_description.native_unit_of_measurement
        return None
```

Just to be sure, is there (in my implementation) something that may I change?

Reply:
There is no fallback to entity description. If the attr exist, that is what will always be used. If you need fallback you need a property to check if the attribute is none. The standard check is if it exists at all.

Reply:
Thanks Joakim, I appreciate your comments.
But I have some doubts if this part is going the way I think.

My intention is to have a 'default' unit of measurement for every `sensor_type`, but the user could always to define the  right one for its device.
For example, a temperature sensor would be 'always' in celsius, but the user must be able to device it in fahrenheit if the device returns this UOM. 

Sorry for asking you about this, but I can't find something similar in the code and that part worries to me.

Reply:
Then keep the property for native unit of measurement, if the attribute is none none, return that, if it is none retunr the value from the description.